btx/diagnostics/ag_behenate.py

```diff
@@ -1,15 +1,39 @@
 import numpy as np
-from btx.misc.radial import radial_profile, q2pix, pix2q
+from btx.misc.radial import *
 import matplotlib.pyplot as plt
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 import matplotlib.colors as colors
 from matplotlib.colors import LogNorm
 
 class AgBehenate:
     
-    def __init__(self):
+    """
+    Class for optimizing geometry (distance to detector and center) from a silver 
+    behenate powder, leveraging the fact that the peaks are equidistant in q-space.
+    """
+    
+    def __init__(self, powder, mask, pixel_size, wavelength):
+        """
+        Instantiate object.
+                
+        Parameters
+        ----------
+        powder : numpy.ndarray, 2d
+            powder diffraction image, in shape of assembled detector
+        mask : numpy.ndarray, 2d
+            binary mask in shape of powder image
+        pixel_size : float
+            detector pixel size in mm
+        wavelength : float
+            beam wavelength in Angstrom
+        """
         self.q0 = 0.1076 # |q| of first peak in Angstrom
-        self.delta_qs = np.arange(0.015,0.05,0.00005) # q-spacings to scan over, better in log space?
+        self.delta_qs = np.arange(0.015,0.05,0.00005) # q-spacings to scan over
```

Comment:
Should we change the default range to something that starts at lower q? Mayb `np.arange(0.005, 0.05, 0.00005)` or would that result too often into the half-deltaQ case?